src/main/java/client/command/commands/gm0/AddDexCommand.java

```diff
@@ -0,0 +1,17 @@
+package client.command.commands.gm0;
+
+import client.Client;
+import client.command.StatCommand;
+
+public class AddDexCommand extends StatCommand {
+
+    {
+        setStat(Stat.DEX);
+        setDescription("Assigns AP into " + stat);
+    }
+
+    @Override
+    public void execute(Client client, String[] params) {
+        super.execute(client, params);
+    }
```

Comment:
What's the point of these 4 identical `execute` implementations in all the subclasses? It's already implemented in `StatCommand`.